### PR TITLE
Rewrite delete room dialog

### PR DESCRIPTION
**Pull request by giuseppeg**
ubergrape/grape-web-client/issues/355

### CODE REVIEW

File: src/components/room-delete/roomDeleteDialogTheme.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    fill: red
+  },
+  messageWarning: {
+    color: red
+  },
+  inputContainer: {
+    padding: [20, 0]
+  },
+  deleteButton: {
+    // FIXME once https://github.com/callemall/material-ui/issues/6227 is soved
```
Comment (kof) [inline src/components/room-delete/roomDeleteDialogTheme.js]: what about this one? I think local package override is better than this

Comment (giuseppeg) [inline src/components/room-delete/roomDeleteDialogTheme.js]: as I said MUI will install and use the local dependency `^0.9.0` even if I explicitly define `^0.10.0` in our repo

**Comment by kof** [comment]
@giuseppeg please resolve the pr and make shrinkwrap, insertion point should work now

File: src/components/room-info/RoomInfo.js [hunk 2]
```diff
@@ -184,29 +187,35 @@ export default class RoomInfo extends PureComponent {
...
                   onClick={this.onInvite}
-                  className={classes.buttonInvite}>
-                    <FormattedMessage
-                      id="inviteMoreToGroup"
-                      defaultMessage="Invite more people to this group" />
+                  className={classes.buttonInvite}
+                >
+                  <FormattedMessage
+                    id="inviteMoreToGroup"
+                    defaultMessage="Invite more people to this group"
```
Comment (kof) [inline src/components/room-info/RoomInfo.js]: description is missing

File: src/components/room-info/RoomInfo.js [hunk 3]
```diff
@@ -184,29 +187,35 @@ export default class RoomInfo extends PureComponent {
...
               <li className={classes.actionItem}>
                 <button
                   onClick={this.onAddIntegration}
-                  className={classes.buttonIntegration}>
+                  className={classes.buttonIntegration}
+                >
                   <FormattedMessage
                     id="addServiceIntegration"
-                    defaultMessage="Add service integration" />
+                    defaultMessage="Add service integration"
```
Comment (kof) [inline src/components/room-info/RoomInfo.js]: description

**Pull request merged**